cpp/dolfinx/geometry/utils.h

```diff
@@ -310,10 +311,13 @@ void _compute_collisions_point(const geometry::BoundingBoxTree<T>& tree,
 {
   std::deque<std::int32_t> stack;
   std::int32_t next = tree.num_bboxes() - 1;
+  std::span<const T> bbox_coordinates = tree.bbox_coordinates();
+  auto view_bbox = [&bbox_coordinates](std::size_t node)
```

Comment:
I don't think this is needed - use [subspan](https://en.cppreference.com/w/cpp/container/span/subspan).